### ISSUE TITLE
Velocity on shaded relief/ 3d topography background plotting error 

### ISSUE THREAD

**Issue by anandsar01** [alexisInSAR/EZ-InSAR#107]
I am the following error while plotting the velocity on either 3 topography or shaded relief background.
![ps_load_dem](https://github.com/user-attachments/assets/ffbe96b5-39ff-48ba-b199-451003322166)

Please let me know how to plot velocity on shaded relief background.

**Comment by alexisInSAR** [comment]
Hi @anandsar01, 

Can you give me the list of files in the "INSAR_XXXXXXXX" directory? 

Kind regards, 

Alexis

**Comment by anandsar01** [comment]
Please the list of files in the InSAR_XXXXXXXX directory 

[ls_InSAR.odt](https://github.com/user-attachments/files/16491083/ls_InSAR.odt)

**Comment by anandsar01** [comment]
Waiting to hear from you @alexisInSAR .

**Comment by alexisInSAR** [comment]
It seems that there is an error with the APS computation. 

Is the command 'ps_plot('v-do') usable? 

Sorry for my late reply. 

Kind regards, 

Alexis

**Comment by anandsar01** [comment]
Yes! I am able to plot results with backgrounds such as 4,5,6 other than shaded relief and 3D topo (2&3) 

Kind regards
Anand
